azureeyemodule/app/iot/iot_interface.cpp

```diff
@@ -147,62 +149,104 @@ static inline std::unordered_map<std::string, std::string> parse_onvif_message(c
     // This naive format works for something as simple as this for now, but we should really consider
     // using a better serialization/deserialization mechanism.
 
-    const std::string delimiters = "-_";
+    const std::string delimiter1 = "-"; // the delimiter to seperate different feature
```

Comment:
As we are no longer calling any of this deserialization stuff, let's remove it all.